stdlib/Dates/src/deprecated.jl

```diff
@@ -68,3 +68,19 @@ end
 
 @deprecate argerror(msg::String) ArgumentError(msg) false
 @deprecate argerror() nothing false
+
+@deprecate round(dt::TimeType,  p::Period, r::RoundingMode{:NearestTiesUp}) round(p, dt, r)
+
+@deprecate round(x::TimeType, p::Period) round(p, x)
+
+@deprecate floor(x::TimeTypeOrPeriod, p::Type{P}) where P <: Period floor(p, x)
+
+@deprecate ceil(x::TimeTypeOrPeriod, p::Type{P}) where P <: Period ceil(p, x)
+
+@deprecate floor(::Type{Date}, x::TimeTypeOrPeriod, ::Type{P}) where P <: Period floor(oneunit(P), Date(x))
+
+@deprecate ceil(::Type{Date}, x::TimeTypeOrPeriod, ::Type{P}) where P <: Period ceil(oneunit(P), Date(x))
+
+@deprecate round(x::TimeTypeOrPeriod, ::Type{P}, r::RoundingMode=RoundNearestTiesUp) where P <: Period round(oneunit(P), x, r)
+
+@deprecate round(::Type{Date}, x::TimeTypeOrPeriod, ::Type{P}, r::RoundingMode=RoundNearestTiesUp) where P <: Period round(oneunit(P), Date(x), r)
```

Comment:
```suggestion
@deprecate round(dt::TimeType,  p::Period, r::RoundingMode{:NearestTiesUp}) round(p, dt, r)
@deprecate round(x::TimeType, p::Period) round(p, x)
@deprecate floor(x::TimeTypeOrPeriod, p::Type{P}) where P <: Period floor(p, x)
@deprecate ceil(x::TimeTypeOrPeriod, p::Type{P}) where P <: Period ceil(p, x)
@deprecate floor(::Type{Date}, x::TimeTypeOrPeriod, ::Type{P}) where P <: Period floor(oneunit(P), Date(x))
@deprecate ceil(::Type{Date}, x::TimeTypeOrPeriod, ::Type{P}) where P <: Period ceil(oneunit(P), Date(x))
@deprecate round(x::TimeTypeOrPeriod, ::Type{P}, r::RoundingMode=RoundNearestTiesUp) where P <: Period round(oneunit(P), x, r)
@deprecate round(::Type{Date}, x::TimeTypeOrPeriod, ::Type{P}, r::RoundingMode=RoundNearestTiesUp) where P <: Period round(oneunit(P), Date(x), r)
```
I think it's clearer to remove the blank lines between these since they're all closely related.

Reply:
@StefanKarpinski I committed these changes